.github/workflows/scripts/build-static.sh

```diff
@@ -2,8 +2,9 @@
 set -euo pipefail
 
 cd MUXSDKStats
+pod cache clean --all
 pod repo update
-pod deintegrate && pod install
+pod deintegrate && pod install --repo-update
```

Comment:
😮

Reply:
I've no idea why the `pod repo update` right before wasn't up to it